osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -236,11 +242,54 @@ func addPackageInstallRemovalTest(pkgTestSetup []*packageManagementTestSetup, te
 	return pkgTestSetup
 }
 
+func addPackageInstallFromNewRepoTest(pkgTestSetup []*packageManagementTestSetup, testProjectConfig *testconfig.Project) []*packageManagementTestSetup {
+	testName := "packageinstallfromnewrepotest"
+	desc := "test package installation from new package"
+	packageName := "osconfig-agent-test"
+	for _, tuple := range tuples {
```

Comment:
Not new to this review but isn't `tuples` naming the variable according to its type? Maybe `platformPkgManagers` or `distros` or `systems` would be clearer. I had to double check the previous source to remember what these things were.

Reply:
changed